src/JunctionTree.jl

```diff
@@ -425,7 +425,8 @@ function drawTreeAsyncLoop( tree::BayesTree,
                             opt::SolverParams;
                             filepath=joinLogPath(opt,"bt.pdf"),
                             dotreedraw = Int[1;]  )
-  #
+  
+  global global_dotreedraw = dotreedraw
```

Comment:
opt.drawtree

Reply:
This is a quick and dirty way to stop the tree drawing if it fails. I would like to use something like opt.drawtree, it would mean replacing dotreedraw, though.

Reply:
okay, sure -- hacks are fine, as long as there is a TODO comment or something to point out that it needs to be fixed somehow.  THansk!

Reply:
Its removed